docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -1,118 +1,117 @@
 # Quick Start Guide
 
-Get ready to get Strapi up and running in **less than 5 minutes** 🚀
+Get ready to get Strapi up and running in **less than 5 minutes** 🚀.
 
 <div class="video-container">
 <iframe width="853" height="480"  src="https://www.youtube.com/embed/_qlLobVjd9k" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
 _For a step-by-step guide, please take a look at the [detailed tutorial](quick-start-tutorial.html)._
 
-## 1. Install Strapi globally
+(Before continuing, please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.)
 
-Please make sure [Node.js and npm are properly installed](install-requirements.html) on your machine.
+## 1. Install Strapi and Create a new project
 
 ```bash
-npm install strapi@beta -g
+yarn create strapi-app my-project --quickstart
 ```
 
-## 2. Create a new project
+**or**
+
+- Use **npm/npx** to install the Strapi project.
 
 ```bash
-strapi new my-project --quickstart
+npx create-strapi-app my-project --quickstart
 ```
 
-## 3. Create an Administrator user
+## 2. Create an Administrator user
 
 Navigate to [**http://localhost:1337/admin**](http://localhost:1337/admin).
 
-- Complete the form to create the first Administrator user.
+- Complete the form to create the first **Administrator** user.
 - Click **Ready to start**.
 
-## 4. Create a new Content Type
+## 3. Create a new Content Type
 
 Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
 
-- Click the **"+ Add Content Type"** button.
-- Enter `restaurant`.
-- Click **"+ Add New Field"**
-  - Click the **String** field.
-  - Type `restaurant` under the **BASE SETTINGS** tab, in the **Name** field.
-  - Check `Required field` and `Unique field` under the **ADVANCED SETTINGS** tab.
-  - Click **Continue**.
-- Click **"+ Add New Field"**
-  - Click the **Text** field.
-  - Type `description` under the **BASE SETTINGS** tab, in the **Name** field.
-  - Click the **ADVANCED SETTINGS** tab, check **Display as a WYSIWYG**.
-  - Click **Continue**.
-- Click the **Save** button and wait for Strapi to restart.
-
-## 5. Create another new Content type
+- Click the **"+ Add Content Type"** link
+- Enter `restaurant`, and click `Done`
+- A window opens with fields options:
+  - Click the **String** field
+  - Type `name` in the **Name** field
+  - Click over to the **ADVANCED SETTINGS** tab, and check the `Required field` and the `Unique field`
+  - Click the **"+ Add New Field"** button
+  - Click the **Rich Text** field
+  - Type `description` under the **BASE SETTINGS** tab, in the **Name** field
+  - Click `Done`
+- Click the **Save** button and wait for Strapi to restart
+
+## 4. Create another new Content type
```

Comment:
Create another new seems redundant ?

Reply:
I agree. After rereading it wasn't very clear.  I made both steps #3 and #4 more clear. Good catch.